lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -1267,6 +1267,14 @@ static Js::FunctionInfo::Attributes GetFunctionInfoAttributes(ParseNodeFnc * pno
     if (pnodeFnc->IsMethod())
     {
         attributes = (Js::FunctionInfo::Attributes)(attributes | Js::FunctionInfo::Attributes::Method);
+
+        // ES19:14.6.13 - Runtime Semantics: ClassDefinitionEvaluation calls ES19:9.2.10 - MakeConstructor.
```

Comment:
Section numbers move around a lot as new sections get added, but the permalinks are meant to be relatively stable over time, so we generally refer to spec sections as #sec-runtime-semantics-classdefinitionevaluation rather than 14.6.13.